src/frontend/Komponenter/Behandling/TidligereVedtaksperioder/TidligereVedtaksperioderInfo.tsx

```diff
@@ -109,25 +57,31 @@ const TidligereVedtaksperioderInfo: FC<{ tidligereVedtaksperioder: ITidligereVed
     tidligereVedtaksperioder,
 }) => {
     return (
-        <>
-            <DivMedBottomMargin>
-                <TabellTidligereVedtaksperioder
-                    infotrygd={tidligereVedtaksperioder.infotrygd}
-                    sak={tidligereVedtaksperioder.sak}
-                />
-            </DivMedBottomMargin>
+        <Container>
+            <FlexDiv>
+                {mapIkon(VilkårInfoIkon.REGISTER)}
```

Comment:
```suggestion
<Registergrunnlag />
```

Her ville jeg bare brukt ikonet direkte. Det er jo ingenting som skal endre seg